### DISCUSSION TITLE
Deluxe scrambled eggs and human meats

### DISCUSSION THREAD

**Discussion by adamkad1** [CleverRaven/Cataclysm-DDA#47628] [Ideas]
So, deluxe scrambled eggs can use mutant meat, but not cooked creep or cooked mongrel. Do we want it? If so, does anyone have a suggestion for what 'special' names to give to them?

**Comment by anothersimulacrum** [comment]
I don't see any reason why they would not be able to use those meats.

**Reply by adamkad1** [reply]
Still no idea for the name tho

**Reply by anothersimulacrum** [reply]
They don't _need_ special names.

**Reply by Qrox** [reply]
May I suggest `deluxe scrambled eggheads`?

**Reply by adamkad1** [reply]
> May I suggest `deluxe scrambled eggheads`?

sounds good to me mon
